src/components/mainPageBody/MainPageBody.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import './main.css';
+import './mainPageBody.css';
 import '../../pages/main/main.css';
```

Comment:
why do you need this import? If you use styles from this file, please move them to 'mainPageBody.css'